### DISCUSSION TITLE
Adding vehicle to database mid-campaign

### DISCUSSION THREAD

**Discussion by Manthhys** [Vdauphin/HeartsAndMinds#1338] [Question and answer] [answered]
Hi guys !

For our unit new H&M campaign I'm looking into a way to manually add a vehicle to the daatabase while the campaign is on-going, for example to add back a player vehicle that glitched or capture an ennemy vehicle.

Which function could I use for my script ? I was thinking this one but I don't know if I'm on the right path.
https://github.com/Vdauphin/HeartsAndMinds/blob/5310d2b41b947141bbbe19e74826e1be570f28a9/=BTC=co@30_Hearts_and_Minds.Altis/core/fnc/db/saveObjectStatus.sqf

All suggestions welcome, thanks in advance ;)

**Comment by Vdauphin** [comment, marked as answer]
Hello,
> example to add back a player vehicle that glitched

They never disappear, you can find them with Curator thanks to this: (execute server side in debug console)
```sqf
{_x addCuratorEditableObjects [btc_vehicles, false];} forEach allCurators;
```
If you want to add new vehicle mid-game read this: http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#2-in-game

>capture an ennemy vehicle.

Destroy them and use the logistic point to repair them. They will be added to the database

> Which function could I use for my script ?

https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/veh/add.sqf

Cheers

**Reply by Manthhys** [reply]
Thanks, I'm gonna look into it ;)

**Reply by Mr-7** [reply]
I don't seem to be able to add a vehicle to the mission during a campaign using the Eden layers. The respawnable one works fine but the standard just stay in place one doesn't seem to add any vehicles when the server loads up (unless I clear the database and wipe the mission)

I'm trying to add a vehicle with modules attached to it in the editor so just spawning it from admin garage or Zeus won't work. 

Is there a way I can add it to the database without restarting mission?

**Reply by Vdauphin** [reply]
> If you want to add new vehicle mid-game read this: http://vdauphin.github.io/HeartsAndMinds/Add-vehicle-or-boat-to-H&M-system#2-in-game
